walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -110,15 +98,10 @@
   */
   private ImmutableList<TimeRange> findFreeRangesAllAttendees(ImmutableList<TimeRange> sortedBusyRangesForMandatoryAttendees, Collection<Event> events, MeetingRequest request) {
     ImmutableList<TimeRange> sortedBusyRangesForOptionalAttendees = findBusyRanges(request.getOptionalAttendees(), events);
-      
-    ArrayList<TimeRange> busyRangesForAllAttendees = new ArrayList<>();
-    busyRangesForAllAttendees.addAll(sortedBusyRangesForMandatoryAttendees);
-    busyRangesForAllAttendees.addAll(sortedBusyRangesForOptionalAttendees);
-    Collections.sort(busyRangesForAllAttendees, TimeRange.ORDER_BY_START);
-    ImmutableList<TimeRange> immutableBusyRangesForAllAttendees = ImmutableList.<TimeRange>builder() 
-              .addAll(busyRangesForAllAttendees) 
-              .build();
-    ImmutableList<TimeRange> freeRangesForAllAttendees = findFreeRanges(request, immutableBusyRangesForAllAttendees);
+
+    ImmutableList busyRangesForAllAttendees = ImmutableList.sortedCopyOf(TimeRange.ORDER_BY_START, 
```

Comment:
Does this build? I believe it should be `ImmutableList<TimeRange>`

Reply:
looking back, im confused why this works too. But it does build!